index.js

```diff
@@ -12,11 +12,11 @@ const DOPPLER_META = ["DOPPLER_PROJECT", "DOPPLER_CONFIG", "DOPPLER_ENVIRONMENT"
 const DOPPLER_TOKEN = core.getInput("doppler-token", { required: true });
 core.setSecret(DOPPLER_TOKEN);
 
-const IS_PERSONAL_TOKEN = DOPPLER_TOKEN.startsWith("dp.pt.");
-const DOPPLER_PROJECT = IS_PERSONAL_TOKEN ? core.getInput("doppler-project") : null;
-const DOPPLER_CONFIG = IS_PERSONAL_TOKEN ? core.getInput("doppler-config") : null;
-if (IS_PERSONAL_TOKEN && !(DOPPLER_PROJECT && DOPPLER_CONFIG)) {
-  core.setFailed("doppler-project and doppler-config inputs are required when using a Personal token");
+const IS_SA_TOKEN = DOPPLER_TOKEN.startsWith("dp.sa.");
+const DOPPLER_PROJECT = IS_SA_TOKEN ? core.getInput("doppler-project") : null;
+const DOPPLER_CONFIG = IS_SA_TOKEN ? core.getInput("doppler-config") : null;
+if (IS_SA_TOKEN && !(DOPPLER_PROJECT && DOPPLER_CONFIG)) {
+  core.setFailed("doppler-project and doppler-config inputs are required when using a Service Account token");
```

Comment:
Should we add a failure message to reject personal tokens entirely? If not, we probably still want this message showing up for those.

Reply:
We don't need to block them outright, but we also don't want to encourage their use.

Reply:
Got it. We probably still want to print an error message. Maybe something like:
```
You're using a Personal token. Consider using a Service Account token instead (they're more secure)! Either way, doppler-project and doppler-config inputs are required when using Personal or Service Account tokens.
```